.github/workflows/build.yml

```diff
@@ -0,0 +1,54 @@
+name: Build
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
```

Comment:
Any reason you aren't building other branches? i.e. feature branches

Reply:
No particular reason, I think I simply copied that from a template without thinking too much about it. I've simplified it to just `on: [ push, pull_requests ]` without mentioning any specific branch names.